pom.xml

```diff
@@ -59,6 +59,7 @@
     <module>package</module>
     <module>shims</module>
     <module>gluten-ras</module>
+    <module>gluten-paimon</module>
```

Comment:
Only include this module when profile paimon is used? Like `gluten-iceberg` module. I assume we should remove this.